### PR TITLE
allow rotation via (yaw,pitch,roll) tuple

### PR DESCRIPTION
**Pull request by greghislop**
simple change avoiding need to create rotation matrix outside Displaz.jl

### CODE REVIEW

File: src/Displaz.jl [hunk 1]
```diff
@@ -572,6 +574,7 @@ viewplot_center_args(s::AbstractString) = ["-viewlabel", string(s)]
 viewplot_center_args(pos) = ["-viewposition", string(pos[1]), string(pos[2]), string(pos[3])]
 # rotation
 viewplot_rotation_args(::Nothing) = []
+viewplot_rotation_args(r::Tuple{Float64,Float64,Float64}) =  vcat("-viewangles", map(i->string(r[i]), 1:3)) #  Set view angles in degrees (yaw, pitch, roll). Equivalent to -viewrotation with rotation matrix R_z(roll)*R_x(pitch-90)*R_z(yaw)
```
Comment (c42f) [inline src/Displaz.jl]:
```suggestion
viewplot_rotation_args(r::Tuple{Float64,Float64,Float64}) =  vcat("-viewangles", string.(r)) #  Set view angles in degrees (yaw, pitch, roll). Equivalent to -viewrotation with rotation matrix R_z(roll)*R_x(pitch-90)*R_z(yaw)
```

**Review by c42f — COMMENTED** [review]
Seems convenient, I guess!

Though in some ways I'd prefer to just clearly document the use of Rotations.jl, because it's more flexible and ultimately might just be clearer. For example, to animate a standard "turntable" rotation about the vertical axis, you can use `Rotations.RotXY`

```julia
using Displaz
using StaticArrays

# Plot some RGB axes
plot3d([SA[rand(), 0, 0] for _=1:1000], color=[1,0,0])
plot3d([SA[0, rand(), 0] for _=1:1000], color=[0,1,0])
plot3d([SA[0, 0, rand()] for _=1:1000], color=[0,0,1])

using Rotations
for θ=0:0.1:2*pi
    viewplot(radius=4, center= 0.2 .* [1,1,1], rotation=RotXY(0.4, θ))
    sleep(0.05)
end
```

**Comment by greghislop** [comment]
Valid point Chris, from my side I saw I had to put in a rotation matrix and thought it's not immediately clear how to generate exactly what I want, I saw how you are simply using the displaz caommand line to pass instructions and looked there seeing I can rotate the way I wanted to. To be honest given this repository provides julia access to displaz it may as well be complete and perform a fuller set of the available operations (or alternative remove them from displaz), even if the rotation matrix instructions were improved.

**Comment by c42f** [comment]
> it may as well be complete and perform a fuller set of the available operations (or alternative remove them from displaz)

Well `-viewrotation` is strictly more general than `-viewangles` so it's more of a convenience for calling displaz from impoverished compute environments like bash, rather than Julia. But I agree it's nice to have the version which just takes Euler angles in degrees for convenience.

It looks like my suggestion to use broadcast rather than `map` has subtly broken this PR. I'll push a fix for that and a couple of other updates here before merging.

**Comment by greghislop** [comment]
thanks Chris

**Comment by c42f** [comment]
@greghislop There's a new release out now with this in it.

**Comment by greghislop** [comment]
I saw, thanks chris

**Pull request merged**